models/articles.models.js

```diff
@@ -14,32 +14,49 @@ exports.selectArticleById = (article_id) => {
     });
 };
 
-exports.selectArticles = (sort_by, order) => {
+exports.selectArticles = (sort_by, order, topic) => {
```

Comment:
If you default the parameters here you would save yourself some if/else blocks down below. For example 'order = 'DESC'', then you can use the default value as long as it passes the checks you are doing, rather than checking if there is a value there and if not - adding DESC as the default